### ISSUE TITLE
Team Goal: Circle back on accessibility course

### ISSUE THREAD

**Issue by awfrancisco** [18F/edu#12]
### Link to document

https://github.com/18F/edu-accessibility
### Detailed information

@gboone and I created some initial modules for the accessibility course, but development has stopped. Those documents, and the open issues in the repo, should be evaluated and prioritized to create an MVP, with the knowledge that this could be a large course. 
### Tasks to completion
- [ ] Evaluate state of current documents
- [ ] Determine MVP level steps
- [ ] Assign team to move forward
- [ ] Connect with #g-accessibility and see if they can offer time/resources
